AIPscan/Aggregator/templates/storage_service.html

```diff
@@ -49,7 +49,16 @@
       <tr>
         <td>{{ mets_fetch_job.download_start }}</td>
         <td>{% if mets_fetch_job.download_end %} {{ mets_fetch_job.download_end - mets_fetch_job.download_start }} {% endif %}</td>
-        <td>{{ mets_fetch_job.total_packages }}</td>
+        <td>{{ mets_fetch_job.total_packages }} packages
+          <i class="fa fa-info-circle fa-lg" id="{{ mets_fetch_job.id }}" style="margin-left: 5px;"></i><br/>
+          <div id="stats-{{ mets_fetch_job.id }}" class="stats" style="font-size:80%; color: #999;">
+            {{ mets_fetch_job.total_aips }} AIPs <br />
+            {{ mets_fetch_job.total_sips }} SIPs <br />
+            {{ mets_fetch_job.total_dips }} DIPs <br />
+            {{ mets_fetch_job.total_replicas }} replicated AIPs <br />
```

Comment:
Nit: We often call these "replicated AIPs", but it's a bit ambiguous whether that means the original or the replica. I think it might be more accurate or at least unambiguous to call them "AIP replicas".

Reply:
+1 Yes agreed.

Reply:
Also, great nit-picking. I do this kind of semantic hair splitting all the time and appreciate it when others do it too. Accuracy is in our field is important. Starting with how we describe the tools that run our systems.